src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -634,9 +634,9 @@ module Make (Inputs : Inputs_intf) :
       in
       go bc2 []
     in
-    Logger.debug t.logger ~module_:__MODULE__ ~location:__LOC__
-      !"Common ancestor: %{sexp: State_hash.t}"
-      ancestor ;
+    Logger.trace t.logger ~module_:__MODULE__ ~location:__LOC__
+      "common ancestor: $state_hash"
```

Comment:
common -> Common (I've been init-capping log messages, but not sure that's a standard)